src/concrete/ml/onnx/ops_impl.py

```diff
@@ -897,7 +899,36 @@ def numpy_equal(
     Returns:
         Tuple[numpy.ndarray]: Output tensor
     """
+    return (numpy.equal(x, y),)
+
+
+def rounded_numpy_equal_for_trees(
+    x: numpy.ndarray,
+    y: numpy.ndarray,
+    *,
+    lsbs_to_remove_for_trees: Optional[int] = None,
+) -> Tuple[numpy.ndarray]:
+    """Compute equal in numpy according to ONNX spec.
```

Comment:
maybe specify that this is for tree-based models only in the docstring as well

Reply:
it s mentioned in lsb argument. 

I added it